Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -1206,12 +1206,12 @@ - (void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)data
 
     Ref<NetworkDataTaskCocoa> protectedNetworkDataTask(*networkDataTask);
     auto downloadID = *networkDataTask->pendingDownloadID();
-    auto& downloadManager = sessionCocoa->networkProcess().downloadManager();
+    CheckedRef downloadManager = sessionCocoa->protectedNetworkProcess()->checkedDownloadManager();
```

Comment:
We should just call `downloadManager()` here. There is no benefit to calling the longer function name. In the coding style we had mentioned that `protectedFoo()` / `checkedFoo()` functions were only to be used to avoid local variables, not to initialize local variables.

Reply:
My proposal:
```
CheckedRef downloadManager = downloadManager = sessionCocoa->networkProcess().downloadManager();
```

Reply:
Oh, right. We shouldn't be calling checked variant here.